feature-libs/quote/components/details/comment/quote-details-comment.component.html

```diff
@@ -14,6 +14,7 @@
       [messageEvents$]="messageEvents$"
       [messagingConfigs]="messagingConfigs"
       (send)="onSend($event, quoteDetails.code)"
```

Comment:
strict issue

Reply:
what exactly? - code is a mandatory property of quote

Reply:
Argument of type '{ files: File | undefined; message: string; itemId?: string | undefined; }' is not assignable to parameter of type '{ message: string; itemId: string; }'.
  Types of property 'itemId' are incompatible.
    Type 'string | undefined' is not assignable to type 'string'.
      Type 'undefined' is not assignable to type 'string'